### DISCUSSION TITLE
Trouble getting library to load in Linux

### DISCUSSION THREAD

**Discussion by Koehn23** [caoccao/Javet#411] [Q&A] [answered]
Caused by: java.lang.UnsatisfiedLinkError: /tmp/javet/1/libjavet-v8-linux-x86_64.v.4.0.0.so: Error loading shared library ld-linux-x86-64.so.2: No such file or directory (needed by /tmp/javet/1/libjavet-v8-linux-x86_64.v.4.0.0.so)

com.caoccao.javet.exceptions.JavetException: Javet library is not loaded because null

I've tried all the linux x86 maven dependencies and I still keep getting this error. Any Suggestions? (This is on an Alipine linux dist., but i would have assumed I would have gotten the OS not supported error code if that was the problem?)

**Comment by caoccao** [comment]
May I know if you were running that in a docker image? The error message indicates `ld-linux-x86-64.so.2` cannot be found. It seems there is something wrong with the Linux distribution.

You may try to run the docker image of my other project JavetShell to get some ideas.

**Reply by Koehn23** [reply]
Yea the image we are running is amazoncorretto:17.0.8-alpine3.18

**Reply by caoccao** [reply, marked as answer]
Please use other base images instead. The image you use seems not to be designed to host native libraries like Javet.
